Source/SwiftLintFramework/Rules/IdentifierNameRule.swift

```diff
@@ -122,15 +121,11 @@ private extension String {
         guard firstCharacter.isUppercase() else {
             return false
         }
+
         guard characters.count > 1 else {
-            return true
+            return !firstCharacter.isLowercase()
         }
         let secondCharacter = String(self[characters.index(after: startIndex)])
         return secondCharacter.isLowercase()
     }
-
-    var isOperator: Bool {
```

Comment:
I wonder if it would be better to just update this function to match the [Swift Language Reference](https://developer.apple.com/library/content/documentation/Swift/Conceptual/Swift_Programming_Language/LexicalStructure.html#//apple_ref/doc/uid/TP40014097-CH30-ID418) (Nate Cook already [translated this to Swift once](https://gist.github.com/natecook1000/c5fb2b8cd0967f53770e)).

Reply:
@sirlantis,

@marcelofabri was right. This property is irrelevant. Operators do not need special treatment.

(But yes, if the property were still useful, it would have been good to change it to use the complete operator list.)